kafka-connect-aws-s3/README-source.md

```diff
@@ -0,0 +1,183 @@
+# AWS S3 Connector Source Configuration
+
+Before reading this document it is recommended to be familiar with the [general readme for the S3 Connectors](README-sink.md).
+
+The primary use case of the source is to read in files written to S3 by the S3 Sink back into Kafka through Kafka Connect.
+
+
+## Source Configuration
+
+An example configuration is provided:
+
+    name=S3SourceConnectorParquet # this can be anything
+    connector.class=io.lenses.streamreactor.connect.aws.s3.source.S3SourceConnector
+    tasks.max=1
+    connect.s3.kcql=insert into $TOPIC_NAME select * from $BUCKET_NAME:$PREFIX_NAME STOREAS `parquet`
+    aws.region=eu-west-1
+    aws.secret.key=SECRET_KEY
+    aws.access.key=ACCESS_KEY
+    aws.auth.mode=Credentials
+    value.converter=io.confluent.connect.avro.AvroConverter
+    value.converter.schema.registry.url=http://localhost:8089
+
+You should replace $BUCKET_NAME, $PREFIX_NAME and $TOPIC_NAME with the names of the bucket, desired prefix and topic.
+
+ACCESS_KEY and SECRET_KEY are credentials generated within AWS IAM and must be set and configured with permissions to write to the desired S3 bucket.
+
+Please read below for a detailed explanation of these and other options, including the meaning of WITH_FLUSH_COUNT and its alternatives.
+
+
+### Source Format configuration
+
+Format configuration is provided by kcql.
+
+The options for json, avro and parquet will look like the below:
+
+    connect.s3.kcql=insert into $TOPIC_NAME select * from $BUCKET_NAME:$PREFIX_NAME STOREAS `JSON`
+
+The options for the formats are case-insensitive, but they are presented here in the most readable form.
+
+#### JSON Input Format Configuration
+
+Using JSON as an input format allows you to read in the JSON-written sink files, line by line.
+
+    STOREAS `JSON`
+
+Please note: The JSON is not parsed by the S3 Source connector.  It is recommended to configure the JsonConverter if you wish the Json to be parsed on entry into Kafka.
+
+    value.converter=org.apache.kafka.connect.json.JsonConverter
```

Comment:
this should use stringconverter otherwise it will wrap the string json in quotes as simple string

Reply:
Json and Text  - there is no difference. unless we support projections in which case yes it makes a difference to parse